src/components/Icon/Expensicons.ts

```diff
@@ -109,7 +110,6 @@ import QuestionMark from '@assets/images/question-mark-circle.svg';
 import ReceiptSearch from '@assets/images/receipt-search.svg';
 import Receipt from '@assets/images/receipt.svg';
 import Rotate from '@assets/images/rotate-image.svg';
-import RotateLeft from '@assets/images/rotate-left.svg';
```

Comment:
@teneeto Is there a reason why this was removed?
This created a regression here
<img width="1512" alt="Screenshot 2024-02-08 at 9 48 09 AM" src="https://github.com/Expensify/App/assets/58412969/4e837b38-b946-4319-aa2a-b9f93c305976">

@marcaaron If you agree, can we handle this in https://github.com/Expensify/App/issues/35663?

Reply:
@ shubham1206agra, I omitted it in a merge conflict. Is there an issue for it I can add it ASAP.

Reply:
Actually, I was thinking of doing it in https://github.com/Expensify/App/issues/35663 since the actual PR was migrated here.

Reply:
Ok, that's brilliant too. whichever way is best, I'm here to assist when you need me. cc @marcaaron

Reply:
Ah good catch, yes let's please fix these regressions as quickly as we can. Thanks!

Reply:
Follow up PR https://github.com/Expensify/App/pull/36144/